### ISSUE TITLE
Enable cancel during interactive tiebreaker

### ISSUE THREAD

**Issue by HEdingfield** [BrightSpots/rcv#132]
Actually enable cancel button for interactive tiebreaker GUI, and also introduce cancel option for CLI version; see:

**Tabulator.java**
`// TODO: If returned eliminatedCandidate is null, infinite loop!`

**TieBreak.java**
```
// TODO: Create and enable cancel option for interactive tiebreaker CLI
// TODO: actually enable cancel button for interactive tiebreaker GUI
```

**Comment by chughes297** [comment]
Seems like we might need this for cert - you can't currently cancel out of a tie-breaker if you select interactive for the tie-breaking mode?

**Comment by HEdingfield** [comment]
Closed with PR #302.
